floss/language/go/extract.py

```diff
@@ -40,6 +43,321 @@ def extract_stackstrings(extract_stackstring_pattern, section_data, min_length)
     return stack_strings
 
 
+VA: TypeAlias = int
+
+
+def get_amd64_lea_xrefs(buf: bytes, base_addr: VA) -> Iterable[VA]:
+    rip_relative_insn_length = 7
+    rip_relative_insn_re = re.compile(
+        b"""
+        (
+              \x48 \x8D \x05  # 48 8d 05 aa aa 00 00    lea    rax,[rip+0xaaaa] 
+            | \x48 \x8D \x0D  # 48 8d 0d aa aa 00 00    lea    rcx,[rip+0xaaaa]
```

Comment:
i dont quite understand this suggestion, can you explain what it fixes?

`48 8D` encodes `lea` and `0D` encodes `rcx`. i think that adding the additional slash would search for a sequence of bytes that includes `x0D` as literal ASCII?

Reply:
that's weird, doesn't seem to be the right fix though

Reply:
let me apologize: you were right - the pattern `\x48 \x8D \x0D` doesn't match the sequence of bytes `| 48 8D 0D |`!

i *think* what happens is that `\x0D` is first decoded into a newline in the string literal, and since we're using `re.verbose` then the newline gets ignored.

there are two fixes:
  1. do like you said, use `\\x0D` which escapes the `\` so that the regex module receives the term `\x0D`, or
  2. use a raw byte literal, like `rb"...."`

i'll probably prefer the second, simply because it keeps the alignment of the lines better, but your note and fix were spot on - im sorry for not understanding your point right away.